legal-api/src/legal_api/services/authz.py

```diff
@@ -44,6 +44,8 @@ class BusinessBlocker(str, Enum):
     # the individual checks within DEFAULT could be added as an individual enum item.
     DEFAULT = 'DEFAULT'
     BUSINESS_FROZEN = 'BUSINESS_FROZEN'
+    DRAFT_PENDING = 'DRAFT_PENDING'
+    GOOD_STANDING = 'GOOD_STANDING'
```

Comment:
the blocker condition we are testing for is not in good standing.  Please rename  `GOOD_STANDING` to `NOT_IN_GOOD_STANDING`